plugins/scaffolder-actions/scaffolder-backend-argocd/package.json

```diff
@@ -33,11 +33,10 @@
     "clean": "backstage-cli clean"
   },
   "dependencies": {
-    "@backstage/backend-common": "^0.24.0",
     "@backstage/config": "^1.2.0",
-    "@backstage/plugin-scaffolder-backend": "^1.24.0",
-    "@roadiehq/backstage-plugin-argo-cd-backend": "^3.0.4",
-    "@backstage/backend-test-utils": "^0.5.0",
+    "@backstage/plugin-scaffolder-node": "^0.4.10",
+    "@backstage/plugin-scaffolder-node-test-utils": "^0.1.11",
```

Comment:
Since this isn't required for the plugin  to run, shouldn't this be in the `devDependencies`?

Reply:
yes. Correct. Good Catch. Thank you